### PR TITLE
rspec tests for sightings api

### PR DESCRIPTION
**Pull request by jeffsmw**
@thaisamarinho can you review this as well 😄

### CODE REVIEW

File: spec/controllers/api/v1/sightings_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::SightingsController, type: :controller do
+  render_views
+
+  let (:pet) { FactoryGirl.create(:pet) }
+
+  describe '#index' do
+    it 'responds with 200 code' do
+      byebug
```
Comment (thaisamarinho) [inline spec/controllers/api/v1/sightings_controller_spec.rb]: remove byebug on line 10

File: spec/controllers/api/v1/sightings_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::SightingsController, type: :controller do
+  render_views
+
+  let (:pet) { FactoryGirl.create(:pet) }
```
Comment (thaisamarinho) [inline spec/controllers/api/v1/sightings_controller_spec.rb]: This could be just :  let(:pet) { create(:pet) }

**Review by thaisamarinho — COMMENTED** [review]
Good job! minor comments here.

**Comment by jeffsmw** [comment]
@whilestevego Changes have been made according to suggestions; ready for review

**Review by whilestevego — APPROVED** [review]
Looks good merge it!

**Pull request merged**